Курсовая работа -1 (home work 10).

zhurlov opened the pull request:
Привет)
listOfEmployees();- не могу нормально вывести в консоль (всех сотрудников). 
И так, Итог курса. Вообще в целом все отлично, а по мне, я посыпался с урока Объектов и Классов. Вроде бы понимаешь теорию, но не можешь связать-писать код самостоятельно. Смотришь на примеры, все становится понятно.
Поэтому пока только базовый вариант решения.
Спасибо!

OlgaLugacheva commented on src/Employee.java:
```diff
@@ -0,0 +1,32 @@
+public class Employee {
+
+    public static int counter = 0;
+    private String FIO;
```
название переменных должно быть с маленькой буквы, даже если это акроним

OlgaLugacheva commented on src/Employee.java:
```diff
@@ -0,0 +1,32 @@
+    }
+
+    public void setEmployeeDepartment() {
+        return;
```
не хватает equlas, hashCode, toString

OlgaLugacheva commented on src/Main.java:
```diff
@@ -1,6 +1,64 @@
+    }
+    private static Employee[] employees = new Employee[10];
 
+    public static void listOfEmployees() {
+        for (Employee employee : employees) {
+            System.out.println(employee);
+        }
```
если переопределить toString то метод будет работать корректно

zhurlov committed:
Курсовая работа -1 (home work 10).
Привет)
listOfEmployees();- не могу нормально вывести в консоль.
И так, Итог курса. Вообще в целом все отлично, а по мне, я посыпался с урока Объктов и Классов.
Вроде бы понимаешь теорию, но не можешь связать-писать код самостоятельно.
Смотришь на примеры, все становится понятно.
Поэтому пока только базовый вариант решения.
Спасибо!

zhurlov committed:
Курсовая работа 1 (home work 10).
Все поправил.
